commons/src/test/java/com/ge/predix/acs/utils/HttpServletRequestUtilTest.java

```diff
@@ -84,6 +85,7 @@ public void getSubdomainEmptyDomain() {
         Assert.assertNull(HttpServletRequestUtil.getSubdomain(hostname, domain));
     }
 
+    @SuppressFBWarnings(value = "NP_NULL_PARAM_DEREF_NONVIRTUAL")
```

Comment:
if we do remove this annotation, we should also remove the "com.google.code.findbugs.annotations" dependency from POMs